machineconfig/master/spec/config/storage/files/keepalived.conf.template L2V0Yy9rZWVwYWxpdmVkL2tlZXBhbGl2ZWQuY29uZi50ZW1wbGF0ZQo=

```diff
@@ -0,0 +1,14 @@
+vrrp_instance VI_1 {
+    state BACKUP
+    interface ${INTERFACE}
+    virtual_router_id 51
+    priority 150
```

Comment:
```suggestion
    priority 40
```
This should be set to less than bootstrap, so that it would be used initially. Script check weight + base priority should be more than bootstrap priority - once masters are running API servers keepalived should switch traffic to them